exporter/awsxrayexporter/translator/cause.go

```diff
@@ -346,3 +349,74 @@ func fillPythonStacktrace(stacktrace string, exceptions []awsxray.Exception) []a
 
 	return exceptions
 }
+
+func fillJavaScriptStacktrace(stacktrace string, exceptions []awsxray.Exception) []awsxray.Exception {
+	r := textproto.NewReader(bufio.NewReader(strings.NewReader(stacktrace)))
+
+	// Skip first line containing top level exception / message
+	r.ReadLine()
+	exception := &exceptions[0]
+	var line string
+	line, err := r.ReadLine()
+	if err != nil {
+		return exceptions
+	}
+
+	exception.Stack = make([]awsxray.StackFrame, 0)
+	for {
+		prefixRegex := regexp.MustCompile(`\s*at (.*?)`)
```

Comment:
You need to make sure to compile regexes as consts at the top-level rather than in logic, or especially in a loop. Compiling regex is very slow.

That being said, are there runtimes that use tabs? I think parsing logic can be strict for known formats, it makes the logic easier to reason about. If stack traces always have spaces, we should just check for spaces.

Reply:
Good point! I raised it to the top-level as a variable, regexp cannot be a constant.

I cannot find a runtime that uses tabs, I changed it to strictly check for spaces.

Reply:
If it's always spaces, why use a regex? Can't you use the `HasPrefix` check similar to the other parsers?